apps/graphql/lib/graphql/schema/user.ex

```diff
@@ -132,6 +132,10 @@ defmodule GraphQl.Schema.User do
       user, _, _ -> {:ok, User.background_color(user)}
     end
 
+    field :demoed, :boolean, resolve: fn
```

Comment:
Can you add documentation here. You can see https://github.com/pluralsh/plural/blob/master/apps/graphql/lib/graphql/schema/repository.ex#L196 for an example.